tests/system/test_active_response/test_netsh_windows_command/test_firewall_status/data/playbooks/configuration.yaml

```diff
@@ -0,0 +1,96 @@
+- name: Configure manager environment
+  hosts: manager
+  become: true
+  tasks:
+
+    - name: Update packages list
+      ansible.builtin.apt:
```

Comment:
` apt`? What happens if we have a RPM manager?

Reply:
Done [here](https://github.com/wazuh/wazuh-qa/pull/3787/commits/e4ded61433bf9f440dd1336be139e4ecb01581ec)